src/features/options/useGetOptions.ts

```diff
@@ -219,7 +219,10 @@ export function useFilteredAndSortedOptions({
 
     // Always remove the rowNode and dataModelLocation at this point. It is only to be used in the filtering
     // process, and will not ruin the comparison later to make sure the state is set in zustand.
-    options = options.map((option) => ({ ...option, rowNode: undefined, dataModelLocation: undefined }));
+    options.map((option) => {
```

Comment:
Nitpick, but since you are now mutating the array instead of reassigning it, is it clearer if you switch to a `.forEach` instead?

Reply:
Good choice. I personally don't see much difference in those two, but I went for a `for (const option of options) {...}` instead.